src/modules/commander/Commander.cpp

```diff
@@ -4339,3 +4350,41 @@ Commander::offboard_control_update(bool &status_changed)
 	}
 
 }
+
+void Commander::esc_status_check()
+{
+	char esc_fail_msg[50];
+	esc_fail_msg[0] = '\0';
+
+	int online_bitmask = (1 << (esc_status.esc_count)) - 1;
+
+	// Check if ALL the ESCs are online
+	if (online_bitmask == esc_status.esc_online_flags) {
+		if (_last_esc_online_flags != esc_status.esc_online_flags) {
+			mavlink_log_info(&mavlink_log_pub, "All ESCs are ready");
```

Comment:
@cmic0 I think we can get rid of this message. No need to tell the user what he does not need to know.